app/src/main/java/com/codesoom/assignment/AppHttpHandler.java

```diff
@@ -0,0 +1,109 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.enums.HttpMethodType;
+import com.codesoom.assignment.error.ClientError;
+import com.codesoom.assignment.service.TaskService;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.util.stream.Collectors;
+
+public class AppHttpHandler implements HttpHandler {
+    TaskService taskService = new TaskService();
```

Comment:
taskService가 재할당되면 곤란하겠죠.

```suggestion
    private final TaskService taskService = new TaskService();
```

Reply:
#### 수정 (c463ec2)
- private인 이유
   - taskService는 AppHttpHandler 내부에서만 사용되는 멤버 변수이기 때문입니다.
- final인 이유
   - taskService가 재할당 될 시 TaskService 인스턴스 변수인 tasks (데이터 목록)가 새로 초기화되기 때문에, 기존 데이터 목록이 날라가는 상황을 방지하기 위해서 final 키워드를 사용해야 합니다.

잘못된 내용이 있으면 피드백 부탁드립니다!

Reply:
꼼꼼하게 답변을 작성하셨네요. 열심히 학습하시는 모습이 멋집니다. 한편 이렇게 정리할 때마다 JLS 문서도 한번씩 읽어두는 습관을 길러보세요.

https://docs.oracle.com/javase/specs/jls/se11/html/jls-6.html#jls-6.6

이렇게 뭔가 찾아볼 일이 있을 때마다 공식 문서를 같이 읽는 것을 10년간 했다고 생각해 봅시다.

Reply:
영어가 어렵지만 공식 문서를 읽는 습관을 들여보겠습니다 :)
감사합니다!